components/automate-ui/src/app/modules/infra-proxy/org-edit/org-edit.component.html

```diff
@@ -30,6 +30,12 @@
         formControlName="admin_key" data-cy="update-org-admin-key"></textarea>   
       </label>
     </chef-form-field>
+    <chef-form-field id='projects-container'>
+      <app-projects-dropdown
```

Comment:
Separate question: On the form shown above, why is the admin key field empty? 
Two possibilities:
(1) It's an oversight; the field should be filled in .
(2) It is deliberately empty for security reasons. If so, there needs to be something to give the user a clue. @susanev, what do you think?

Reply:
@msorens this will be fixed in another issue, the admin key will have its own tab like the user password reset.